packages/opentelemetry-core/src/trace/GlobalTracer.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { NoopTracer } from './NoopTracer';
+
+const noopTracer = new NoopTracer();
+let _globalTracer: types.Tracer | null = null;
+
+export class GlobalTracerDelegate implements types.Tracer {
```

Comment:
This file is called `GlobalTracer`. I've been working to add a linter with the `export-name` rule. Would you be willing to either rename the class or the file to be consistent?

Reply:
I'd prefer to rename the file.